skr/Makefile

```diff
@@ -107,6 +110,12 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: ## NOT SUPPORTED - Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/default | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
+.PHONY: helm-prepare
+helm-prepare: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Comment:
This just creates the yaml for the helm chart, right? And does not deploy anything as mentioned in the comment. Please adapt comment